packages/features/settings/layouts/SettingsLayout.tsx

```diff
@@ -55,6 +55,7 @@ const tabs: VerticalTabItemProps[] = [
       { name: "password", href: "/settings/security/password" },
       { name: "impersonation", href: "/settings/security/impersonation" },
       { name: "2fa_auth", href: "/settings/security/two-factor-auth" },
+      { name: "passkeys", href: "/settings/security/passkeys" },
```

Comment:
This still makes it appear *above* SSO. Is this position fine?

If we want it to be the last entry in the list, will have to add it below

Reply:
positioning is fine